api/applicationserver.proto

```diff
@@ -151,3 +151,32 @@ service AsEndDeviceRegistry {
   };
 }
 
+// MQTTConfig represents the configuration of the Application Server MQTT frontend.
+message MQTTConfig {
```

Comment:
I think it's better to (1) include `ApplicationServer` in here or (2) remove the comment about "Application Server". We'll likely add a similar endpoint to the Gateway Server too.

Reply:
Yes, this is reusable and not AS specific